kyaml/setters2/settersutil/settercreator.go

```diff
@@ -65,7 +65,7 @@ func (c SetterCreator) Create(openAPIPath, resourcesPath string) error {
 		Outputs: []kio.Writer{inout},
 	}.Execute()
 	if a.Count == 0 {
-		return errors.Errorf("created setter doesn't match any fields")
+		fmt.Printf("setter %s doesn't match any field in resources, but creating setter definition\n", c.Name)
```

Comment:
I'm not sure if this is the best way to handle warnings. Ideally I think we want the information to be returned to the command which can then make a decision on how it should be printed. For example, we should make it possible to write to another stream than stdout. Not sure what is the idiomatic way in go to handle this.

Reply:
Agree with Morten; it's either an error to be passed up and eventually handled or alternatively causing the program to exit non-zero (with a logged error), or it's not an error.